collector/otel-setup/newrelic.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: 'Set up the pganalyze collector with New Relic'
+backlink_href: /docs/collector/otel-setup
+backlink_title: 'OpenTelemetry Setup'
+---
+
+You can set up the pganalyze collector OpenTelemetry exporter with New Relic so
+that you can have a span for the EXPLAIN plan inside of the trace of your
+application, such as a web request trace.
+The span contains the link to the EXPLAIN plan in the pganalyze app, so that you
+can easily visit the pganalyze page to find out why the query is slow.
+
+New Relic provides OTLP endpoints that you can use to export tracing spans from
+the pganalyze collector. In your application side, you can either use the
+New Relic agent or OpenTelemetry SDKs.
+
+## Setup steps
+
+1. [Start sending data of your application to New Relic](#start-sending-data-of-your-application-to-new-relic)
+2. [Add traceparent query tag to the query](#add-traceparent-query-tag-to-the-query)
+3. [Create a New Relic API key](#create-a-new-relic-api-key)
+4. [Setup an API key and an endpoint in the pganalyze collector](#setup-an-api-key-and-an-endpoint-in-the-pganalyze-collector)
+
+### Start sending data of your application to New Relic
+
+To start sending data of your application to New Relic, you can either use the
+guided install and install the New Relic agent, or use OpenTelemetry SDKs.
+You can read how to setup OpenTelemetry integration in [their documentation](https://docs.newrelic.com/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-set-up-your-app/).
+
+Using New Relic agents If you choose to use the New Relic agent, make sure to use the version that
+supports [Distributed Tracing](https://docs.newrelic.com/docs/distributed-tracing/concepts/introduction-distributed-tracing/).
+Latest version of agents in each language support it and also it's enabled by
```

Comment:
```suggestion
The latest versions of agents in each language support it and it's enabled by
```
Since there's multiple agents, there's multiple versions.